tests/test_enhanced.py

```diff
@@ -8,3 +19,83 @@ def test_identify_rotatable_bonds_runs_on_freesolv():
 
     for mol in mols:
         _ = identify_rotatable_bonds(mol)
+
+
+@pytest.fixture
+def ff_pair():
+    ff0 = Forcefield.load_from_file(DEFAULT_FF)
+    ff1 = Forcefield.load_from_file(DEFAULT_FF)
+
+    # Modify the charge parameters for ff1
+    for h, p in zip(ff1.get_ordered_handles(), ff1.get_ordered_params()):
+        if isinstance(h, AM1CCCHandler):
+            p += 1.0
```

Comment:
Nit: to avoid second-guessing whether modifying the arrays returned by `ff1.get_ordered_params()` in-place also modifies `ff1` in-place

```suggestion
    ff1.q_handle.params += 1.0
```